packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -1634,6 +1635,9 @@ class SubmenuButton extends StatefulWidget {
   /// An optional icon to display after the [child].
   final Widget? trailingIcon;
 
+  /// An optional arrow icon to display instead of the default one.
+  final Widget? arrowIcon;
+
```

Comment:
It's a `Widget` type so I think a more general name would be appreciated. Users can use any icon/custom widget to open sub-menus, not just an arrow icon.

Reply:
Changed to `submenuIcon`